### PR TITLE
Updating Templates for Beta 1

### PR DESCRIPTION
**Pull request by grokys**

### CODE REVIEW

**Comment by grokys** [comment]
I've converted the `AvaloniaApplicationTemplate` (net461) to the new project system. @kekekeks is the any reason to have separate netfx and netcore project templates any more, given that the only difference should be the `TargetFramework`?

**Comment by kekekeks** [comment]
I was mostly concerned about the possibility of new SDK template not working (you know, in May nothing related to the new SDK was working quite right), but now we can remove the old one.

It would be nice to be able to select the target framework (net461/netcoreapp2.0) from the project creation wizard, but I don't how to do that.

**Comment by jp2masa** [comment]
Note: Upgrading `Microsoft.VSSDK.BuildTools` from `15.0.26201` to `15.5.100` limits the Visual Studio version to 15.5+ (for building). The same happens for other Visual Studio SDK packages, if they are libraries (not build tools), it will limit the Visual Studio version for extension consumers too. Anyway, only build tools are being updated here, and most VS 2017 users frequently update VS I think.
Also, I think that project templates (at least the ones using the new project system) should have `PromptForSaveOnCreation` set to `true` (see #51).

File: src/Templates/AvaloniaApplicationTemplate/nuget.config [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
```
[the remark this answers is not in the document]
Comment (grokys) [reply src/Templates/AvaloniaApplicationTemplate/nuget.config]: I was thinking we'd just update the package version and remove this file when we release 0.6.0. This is mainly for testing at the moment.

**Pull request merged**